docs/literate/intro/speed.jl

```diff
@@ -0,0 +1,123 @@
+# # Speed measurement
+# Here we will compare the speed of plotting UnfoldMakie with MNE (Python) and EEGLAB (MATLAB).
+#
+# Three cases are measured: 
+# - Single topoplot
+# - Topoplot series with 50 topoplots
+# - Topoplott animation with 50 timestamps
+
+using UnfoldMakie
+using TopoPlots
+using PyMNE
+using PythonPlot
+using BenchmarkTools
+using Observables
+using CairoMakie
+using FFMPEG_jll
+
+# Data input 
+dat, positions = TopoPlots.example_data()
+df = UnfoldMakie.eeg_array_to_dataframe(dat[:, :, 1], string.(1:length(positions)));
+
+# # Topoplots
+
+# UnfoldMakie.jl
+@benchmark plot_topoplot(dat[:, 320, 1]; positions = positions)
+
+# UnfoldMakie.jl with DelaunayMesh
+@benchmark plot_topoplot(
```

Comment:
I've seen that there is little difference, maybe remove delaunay mesh here?

Reply:
300 ms
Good question, we can remove it or also inlude mne with linear interpolation